### DISCUSSION TITLE
how to apply energetic threshold to an order parameter

### DISCUSSION THREAD

**Discussion by RWTHLHK** [idaholab/moose#27165] [Q&A Modules: Phase field] [answered]
I am deriving a new phase field model. The order parameter is associated with a generalized energetic force Y, the order parameter can only evolve when this energetic force Y >= 0(which is common in thermodynamics), how can I add this as a threshold function?

**Comment by GiudGiud** [comment]
@laagesen @amjokisaari

**Reply by laagesen** [reply, marked as answer]
You could make the order parameter mobility a material property that includes a conditional statement that has Y>0, I think you can do this with a DerivativeParsedMaterial

**Reply by RWTHLHK** [reply]
Genius idea!
